cmd/user-cluster-controller-manager/main.go

```diff
@@ -59,33 +61,34 @@ import (
 )
 
 type controllerRunOptions struct {
-	metricsListenAddr     string
-	healthListenAddr      string
-	version               string
-	networks              networkFlags
-	namespace             string
-	clusterURL            string
-	clusterName           string
-	openvpnServerPort     int
-	kasSecurePort         int
-	tunnelingAgentIP      flagopts.IPValue
-	overwriteRegistry     string
-	cloudProviderName     string
-	nodelabels            string
-	seedKubeconfig        string
-	ownerEmail            string
-	updateWindowStart     string
-	updateWindowLength    string
-	dnsClusterIP          string
-	nodeLocalDNSCache     bool
-	opaIntegration        bool
-	opaWebhookTimeout     int
-	useSSHKeyAgent        bool
-	caBundleFile          string
-	mlaGatewayURL         string
-	userClusterLogging    bool
-	userClusterMonitoring bool
-	ccmMigration          bool
+	metricsListenAddr                                       string
+	healthListenAddr                                        string
+	version                                                 string
+	networks                                                networkFlags
+	namespace                                               string
+	clusterURL                                              string
+	clusterName                                             string
+	openvpnServerPort                                       int
+	kasSecurePort                                           int
+	tunnelingAgentIP                                        flagopts.IPValue
+	overwriteRegistry                                       string
+	cloudProviderName                                       string
+	nodelabels                                              string
+	seedKubeconfig                                          string
+	ownerEmail                                              string
+	updateWindowStart                                       string
+	updateWindowLength                                      string
+	dnsClusterIP                                            string
+	nodeLocalDNSCache                                       bool
+	opaIntegration                                          bool
+	opaWebhookTimeout                                       int
+	useSSHKeyAgent                                          bool
+	caBundleFile                                            string
+	mlaGatewayURL                                           string
+	userClusterLogging                                      bool
+	userClusterMonitoring                                   bool
+	userClusterPrometheusCustomScrapeConfigsConfigMapPrefix string
```

Comment:
Maybe it would be possible to find a somewhat shorter name for this.

Reply:
Naming is hard :)
Just shortened it to `prometheusScrapeConfigMapPrefix`